images/router/haproxy/pod.json

```diff
@@ -45,6 +45,10 @@
           {
             "name": "OPENSHIFT_INSECURE",
             "value": "${OPENSHIFT_INSECURE}"
+          },
+          {
+            "name": "DROP_SYN_DURING_RESTART",
+            "value": "0"
```

Comment:
aah ok. That makes sense. I assumed you meant to enable it by default, so my comment was specific to where this got added/set.